src/components/Content/ProjectContent/ExperimentsTabs/_/index.js

```diff
@@ -21,17 +21,40 @@ const ExperimentsTabs = ({
   handleChange,
   handleMoveTab,
   activeExperiment,
+  deleteHandler,
 }) => {
   // COMPONENTS RENDERS
+  // CONTEXT MENU
+  const menu = (experimentId) => (
+    <Menu>
+      <Menu.Item key='1'>
+        <Icon type='edit' />
+        Renomear
+      </Menu.Item>
+      <Menu.Item key='2'>
+        <Popconfirm
+          title='Você tem certeza que deseja excluir esse projeto?'
+          onConfirm={() => deleteHandler(experimentId)}
+          okText='Sim'
+          cancelText='Não'
+        >
+          <Icon type='delete' />
+          Excluir
+        </Popconfirm>
+      </Menu.Item>
+    </Menu>
+  );
```

Comment:
@hugooliv4 @SchafferCPqD este PR era só pra parte de "Muitos cliques pra sair do projeto", certo?
Se menu de contexto for em outro PR, é legal retirar o código desta branch.